app/views/shared/_user_util_links.html.erb

```diff
@@ -12,7 +12,7 @@
 
 <ul class="navbar-nav">
   <li class="nav-item">
-    <%= link_to 'Spotlight at Stanford', main_app.root_path, class: 'nav-link fw-semibold' %>
+    <%= link_to 'Spotlight at Stanford', main_app.root_path, class: 'nav-link emphasized' %>
```

Comment:
@alundgard requested 600 in https://github.com/sul-dlss/exhibits/issues/2722, but this PR changes it to 500.  Is that right?

Reply:
Yes, the #2742 design supersedes #2722.